api/message.ts

```diff
@@ -48,8 +48,9 @@ const message = createApiMessageHandler({
     mockRoamAlphaAPI({ token: accessToken, graph: workspace });
     return notebookRequestHandler;
   },
-  getNotebookResponseHandler: (token) => async (response) => {
+  getNotebookResponseHandler: () => async (response) => {
     // TODO
+    console.log("getNotebookResponseHandler", response);
```

Comment:
TODO - once App 2 has a response ready, it will hit this `getNotebookResponseHandler`. This needs to make the data accessible somehow to the `backendCrossNotebookRequests` method below...

Reply:
I think we'll want to open a super short lived websocket and use _that_ for the connectionId?

Like we may, in the short term, replicate Server-Sent events, but as websockets. And also maintaining client sessions may be useless, as it could be duplicative with what Clerk's already doing